### ISSUE TITLE
The text in the menubar is a bit hard to read and not all the HTML elements are colored.

### ISSUE THREAD

**Issue by bound-variable** [faelv/dark-taste-theme#1]
Themes designed for HTML usually color the **&lt** and **&gt** parts inside of the **code** elements.

And for me, the text in the menubar is a little hard to read. Of course, this is going to vary with each user because of differences with their eyes and their monitors.

Thanks for your work. Cheers.
![dark taste](https://user-images.githubusercontent.com/51193876/122268395-de974480-ce90-11eb-907c-af9f39378530.jpg)

**Comment by faelv** [comment]
@ConsciousClay  I see now... thanks for letting me know. What color you consider the best?

- Red
![red](https://user-images.githubusercontent.com/33964313/122276341-11eec900-cebb-11eb-927e-cef2a1d7ed4c.png)
- Light purple
![light-purple](https://user-images.githubusercontent.com/33964313/122276484-35b20f00-cebb-11eb-800e-fe27f25bdbc3.png)
- Light yellow
![light-yellow](https://user-images.githubusercontent.com/33964313/122276527-41053a80-cebb-11eb-9acc-894c5084241b.png)

Now about the menus, they seem fine to me but as you said it varies per person, you probably want to increase the contrast or use pure white, to do so, in your `settings.json` file add the following lines and customize the hex values to your needs:

```json
"workbench.colorCustomizations": {
        "[Dark Taste]": {
            "tab.activeForeground": "#cdcccb",
            "tab.inactiveForeground": "#cdcccb99",
            "activityBar.foreground": "#cdcccb",
            "menu.background": "#2a2928",
            "menu.foreground": "#cdcccb",
            "menubar.selectionForeground": "#cdcccb"
        }
    }
```

You can do that to any theme if don't like a particular color, more info here: https://code.visualstudio.com/api/references/theme-color

**Comment by bound-variable** [comment]
Red looks best to me. But I'm color blind, so you should judge for yourself
or ask someone else.

Thanks for your concern.

**Comment by faelv** [comment]
Closed by 4fb8c51541784617229b96431dbb0dfbb176c59e
